pkg/apiserver/apiserver.go

```diff
@@ -113,8 +113,11 @@ const (
 
 // InstallREST registers the REST handlers (storage, watch, proxy and redirect) into a restful Container.
 // It is expected that the provided path root prefix will serve all operations. Root MUST NOT end
-// in a slash. A restful WebService is created for the group and version.
-func (g *APIGroupVersion) InstallREST(container *restful.Container) error {
+// in a slash.
+// If you are installing the API group to an existing webservice then it should be passed in.  If
```

Comment:
What do you think about adding a separate Update or Reinstall method, instead of making this method do two things? Perhaps it could even find the correct web service inside the container? Is there a reason to make the caller find the web service?